tests/Feature/Controllers/Api/Journals/Modules/Weather/WeatherControllerTest.php

```
@@ -0,0 +1,177 @@
+        $this->assertEquals('heavy', $entry->moduleWeather->precipitation);
+        $this->assertEquals('very_short', $entry->moduleWeather->daylight);
+    }
+
+    #[Test]
+    public function it_validates_condition_must_be_valid(): void
+    {
+        $user = User::factory()->create();
+        $journal = Journal::factory()->create([
+            'user_id' => $user->id,
+        ]);
+        $entry = JournalEntry::factory()->create([
```

PHPMD (2.15.0)

Avoid unused local variables such as '$entry'. (undefined)

(UnusedLocalVariable)